README.md

```diff
@@ -22,6 +22,29 @@ A user should be able to interact with the echo server as follows:
 ## Setup
 1. [Clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your local computer.
 2. Use your terminal to navigate into your new http_server folder and install project dependencies by running `mix compile`.
-3. To start the Echo Server, call `mix start` from within the http_server folder.
-4. To connect to the Server as a client, [install telnet](https://formulae.brew.sh/formula/telnet) and call `telnet 127.0.0.1 4040` in a local terminal.
-5. To run the tests, call `mix test` from within the http_server folder.
+3. You may notice that the `test/http_server_spec` submodule exists, but has no files. Call `git submodule init` and `git submodule update` from within the http_server folder to initialize your local git modules configuration file and fetch all data from the [http_server_spec](https://github.com/8thlight/http_server_spec) project.
```

Comment:
this did not work for me.
<img width="895" alt="image" src="https://github.com/kelseyroy/http_server/assets/26009042/76103a17-f5db-4256-b152-1db7ea3032ea">

Reply:
Hmmm if you cd into `test/http_server_spec`, and run `git submodule init` and `git submodule update`, does that work? If not, does that same directory already have files in it?

Reply:
<img width="954" alt="Screenshot 2023-05-30 at 3 55 31 PM" src="https://github.com/kelseyroy/http_server/assets/73620216/e772034a-6912-4510-ac29-22577ad5f8f7">

Set up worked for me!

Reply:
Ahh ok that's probably the problem here. I just updated the README to clarify that you need to navigate into `test/http_server_spec` before initializing the submodule. Thanks @t-keazirian! I also changed the URL in .gitmodules from the SSH link I used to clone the repository to an HTTPS url, just in case that was contributing to the problem.

Reply:
I got it to work this morning, but I had to update the [submodule "test/htp_server_spec"] config to use ssh and not https.

<img width="577" alt="image" src="https://github.com/kelseyroy/http_server/assets/26009042/39a224ed-f65d-4df1-bc8b-e1976e1ed291">

Reply:
Ok good to know! I'll drop the commit that changes this from the PR.